### ISSUE TITLE
Translation file

### ISSUE THREAD

**Issue by killawabbit** [acf-extended/ACF-Extended#16]
**Is your feature request related to a problem? Please describe.**
I would like to be able to translate this plugin, but there's no .pot file (translation template file), no references or links where such would be available (I also googled it). 
**Describe the solution you'd like**
I'd like to see an official .pot file and languages folder for the plugin. Preferably within the Github repo and with the official plugin. If it's available from another source I'd like it announced. 

**Describe alternatives you've considered**
I use the Loco Translation plugin and it is able to create a .pot file, however it doesn't seem to pickup all the text strings used by the plugin as translatable. 

**Additional context**
I've been trying, but haven't managed to figure out exactly why. Maybe not all text strings within the plugin are made translatable correctly?

**Comment by acf-extended** [comment]
Hello @killawabbit,

I saw your pull request for translations on ACF Extended. I created a specific branch for this task. While I'm impressed by your pull request (thanks for the hard work), I will need to manually check everything, as it could potentially break many parts of the plugin.

This task will require some time, as I'm focused on features implementations. I cannot give you an ETA, but be assured that you'll be mentioned in the special thanks of the plugin :)

Regards.
